gwt-material/src/main/java/gwt/material/design/client/ui/MaterialNavBar.java

```diff
@@ -85,6 +92,11 @@ public MaterialNavBar() {
     }
 
     @Override
+	public void clear() {
+    	div.clear();
```

Comment:
You mean all spaces or 2 tabs. I saw that when creating eclipse users spaces and i then used tab.

Whais should it be?